object_parameters/GeoF_ItemBox.json

```diff
@@ -1,22 +1,34 @@
 {
-  "Object Parameters":
-          ["Height Offset",
-            "ItemBox Type",
-            "Unused",
-            "Unused",
-            "Unused",
-            "Unused",
-            "Unused",
-            "Unused"],
-  "Assets": [],
-  "Tooltips": [
-    "How high off the ground the itembox will be. Itembox must be grounded and this set to 135.",
-    "Whether itembox is single or double:\n0 = both\n1 = single only\n3 = double only",
-    "",
-    "",
-    "",
-    "",
-    "",
-    ""
-  ]
+    "Object Parameters": [
+        "Height Offset",
+        "ItemBox Type",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused",
+        "Unused"
+    ],
+    "Assets": [],
+    "Tooltips": [
+        "How high off the ground the itembox will be. Itembox must be grounded and this set to 135.",
+        "Whether itembox is single or double:\n0 = both\n1 = single only\n3 = double only",
+        "",
+        "",
+        "",
+        "",
+        "",
+        ""
+    ],
+    "RouteInfo": "Indiv",
```

Comment:
Question: What's "Indiv"? Individual? (And was there a technical reason for abbreviating it?)

How about "exclusive" as antonym to "shared"?

Reply:
"Indiv" is short for "Individual" because I am bad at spelling and also lazy. "Individual" seemed more natural, and I also think it's (probably) more reflective of what is possible. "Exclusive" may communicate that an object cannot share a route with another object, when that case is probably possible.

Reply:
I'd probably prefer "Individual" over "Indiv". Other options could be "single" or "particular" (although I liked "exclusive" the best). Leaving it to you, though.

Reply:
If the only options are shared or not shared, a boolean could be used: `"SharedRoute"` (boolean) instead of `"RouteInfo"` (string).

Reply:
The third option is not having a route.